.github/workflows/ci-cd-workflow.yml

```diff
@@ -59,6 +59,19 @@ jobs:
       - name: Run tests
         run: mvn test
 
+      - name: Set up analysis JDK
+        uses: actions/setup-java@v3
+        with:
+          cache: maven
+          distribution: adopt
```

Comment:
The action encourages using `temurin` over `adopt` and that keeps it consistent with the distribution used for building: https://github.com/actions/setup-java?tab=readme-ov-file#supported-distributions.

N.B. there is `actions/setup-java@v4` available.

Reply:
Thanks for spotting this. Fixed!